### PR TITLE
Fixed some bugs and layout improvements

### PR DESCRIPTION
**Pull request by tonin-alessandra**
Hi everybody, I open this small pr to fix some big problems on GeoCalendar.

First of all, I fixed two bugs regarding permissions' request:
1) when installing the app for the first time, there was a crash due to maps related permissions
2) there was a bug on READ_CONTACTS permission request, which made the app crashing when adding a contact to the friends' list.

Then I improved a bit the layout of friends' list: I didn't change from TextViews to menu items because I had to change too much code, so I increased text size and added some padding to make contacts more readable and easy to click.

I tried it on the emulator and also on two real phones, and it seemed all ok.
If you can, please try the demo on your devices and tell me if the friends' list layout works properly.

Thank you in advance for your time, and good review!

### CODE REVIEW

File: demo/src/main/java/com/eis/geoCalendar/demo/MainActivity.java [hunk 1]
```diff
@@ -246,18 +244,41 @@ public void onEventTriggered(Event event) {
      * Requests Android permissions if not granted
      */
     public void requestPermissions() {
-        ActivityCompat.requestPermissions(this, LocationManager.getPermissions(), APP_PERMISSION_REQUEST_CODE);
-        ActivityCompat.requestPermissions(this, FriendsListManager.getPermissions(), APP_CONTACTS_PERMISSION_REQUEST_CODE);
```
Comment (turcato-niccolo) [inline demo/src/main/java/com/eis/geoCalendar/demo/MainActivity.java:250]: Ah beh sì scemo io, le richieste dei permessi una dietro l'altra non funzionano, per quello alla seconda apertura chiedeva per i contatti

File: demo/src/main/java/com/eis/geoCalendar/demo/MainActivity.java [hunk 2]
```diff
@@ -246,18 +244,41 @@ public void onEventTriggered(Event event) {
...
+    /**
+     * Callback for permissions' request. If they are granted, unlock all GoogleMaps features.
+     *
+     * @param requestCode  The same code used on permissions request.
+     * @param permissions  The list of requested permissions.
+     * @param grantResults The results of each permission's request.
+     * @author Alessandra Tonin
+     */
+    @Override
+    public void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
```
Comment (turcato-niccolo) [inline demo/src/main/java/com/eis/geoCalendar/demo/MainActivity.java:272]: Ottimo

**Review by bortoletti-giorgia — APPROVED** [review]

**Review by turcato-niccolo — APPROVED** [review]
Looks good to me

File: demo/src/main/java/com/eis/geoCalendar/demo/MainActivity.java (hunk 1 above)
Comment (tonin-alessandra) [reply demo/src/main/java/com/eis/geoCalendar/demo/MainActivity.java:250]: Io non lo sapevo proprio invece ahah

**Review by francescobau — APPROVED** [review]
I approve your changes.

**Pull request merged**